polaris-core/src/main/java/org/apache/polaris/core/PolarisCallContext.java

```diff
@@ -85,6 +91,10 @@ public Clock getClock() {
     return clock;
   }
 
+  public String getRequestId() {
```

Comment:
I'm not thrilled about the idea of adding more stuff to this class, which is already a bag of unrelated beans. 

Also, we already have the notion of a request ID for logging purposes:

https://github.com/apache/polaris/blob/ab228afa4d975faabb7aaf1e8abb0804f5b9d353/runtime/service/src/main/java/org/apache/polaris/service/quarkus/logging/QuarkusLoggingConfiguration.java#L28

I would prefer to use the same request ID coming from HTTP headers if available, and if not, use a random UUID. Wdyt?